Answers.md

```diff
@@ -0,0 +1,21 @@
+1# -    What is the difference between an adaptive website and a fully responsive website?
+
+    An adapative website is a website that looks good on all devices. But a fully responsive website is a website that responds correctly on all screen sizes with 100% widths.
+
+2# -    Describe what it means to be mobile first vs desktop first.
+
+    Mobile first uses min-width and starts at a smaller breakpoint and goes higher for bigger devices.
+    Desktop first uses max-width and starts a at the biggest breakpoint and gets smaller as the screen shrinks.
+
+3# -    What does font-size: 62.5% in the html tag do for us when using rem units?
```

Comment:
More specifically, it causes the default font-size for the page to be sent to 10px, which lets us use rems more easily